2025-09-15 15:24:31 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:24:31 [INFO] core: Saving tenant proliance with schema: proliance, Org ID: TEN-0003
2025-09-15 15:24:31 [INFO] core: Tenant updated: proliance for tenant proliance
2025-09-15 15:24:35 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:24:35 [INFO] core: Saving tenant proliance with schema: proliance, Org ID: TEN-0003
2025-09-15 15:24:35 [INFO] core: Tenant updated: proliance for tenant proliance
2025-09-15 15:24:41 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:24:41 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:24:46 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:24:46 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:26:40 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:26:40 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:27:03 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:27:03 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:12 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 15:30:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:30 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:30 [INFO] core: Saving tenant proliance with schema: proliance, Org ID: TEN-0003
2025-09-15 15:30:45 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 15:30:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:55 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:55 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:55 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:55 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:56 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:56 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:56 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:56 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:56 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 15:30:56 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:56 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:56 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:30:56 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:30:56 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 15:31:01 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:01 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:16 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:16 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:32 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:32 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:41 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:41 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:31:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:31:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:02 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:02 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:12 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:12 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:32 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:32 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:32:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:32:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:33:02 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:33:02 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:33:12 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:33:12 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:33:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:33:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:33:32 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:33:32 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:33:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:33:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:34:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:34:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:35:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:35:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:36:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:36:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:36:54 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:36:54 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:37:02 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:37:02 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:37:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:37:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:37:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:37:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:39:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:39:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:40:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:40:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:41:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:41:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:42:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:42:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:44:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:44:06 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:44:25 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:44:25 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:44:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:44:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:44:47 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 15:45:03 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:45:03 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:45:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:45:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:46:09 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 15:46:13 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 15:46:23 [DEBUG] core: Tenant from user: proliance
2025-09-15 15:46:23 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 15:46:42 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 15:46:43 [ERROR] django.request: Internal Server Error: /api/token/refresh/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/auth_service/views.py", line 231, in post
    payload = decode_rsa_jwt(refresh_token)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/utils/jwt_rsa.py", line 112, in decode_rsa_jwt
    keypair = RSAKeyPair.objects.filter(kid=kid, active=True).first()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1104, in first
    for obj in queryset[:1]:
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 384, in __iter__
    self._fetch_all()
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 515, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 486, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/views.py", line 303, in post
    UserActivity.objects.create(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 663, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 902, in save
    self.save_base(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1008, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1169, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1210, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1868, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1882, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^

2025-09-15 15:55:37 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 16:20:01 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 16:20:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:08 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:08 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:08 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:08 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:11 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:11 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:12 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:20:12 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:20:16 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:16 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:36 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:36 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:46 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:46 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:20:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:20:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:21:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:21:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:21:16 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:21:16 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:21:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:21:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:21:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:21:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:21:46 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:21:46 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:22:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:22:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:23:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:23:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:13 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:13 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:24:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:18 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:24:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:38 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:38 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:24:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:24:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:08 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:08 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:38 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:38 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:25:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:25:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:08 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:08 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:38 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:38 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:26:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:26:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:08 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:08 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:38 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:38 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:27:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:27:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:28:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:28:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:15 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:29:15 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:29:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:29:58 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:29:58 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:28 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:28 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:30:58 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:30:58 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:31:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:31:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:31:56 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:31:56 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:31:57 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:31:57 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:31:58 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:31:58 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:31:59 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:31:59 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:02 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:02 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:03 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:03 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:04 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:04 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:05 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:05 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:06 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:06 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:06 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:32:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:06 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:06 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:06 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:06 [INFO] core: Fetched TenantConfig for tenant proliance
2025-09-15 16:32:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:12 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:12 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:12 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:12 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:15 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:15 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:16 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:16 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:17 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:17 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:27 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:27 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:48 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:48 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:32:58 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:32:58 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:33:07 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:33:07 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:33:18 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:33:18 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:33:28 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:33:28 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:33:37 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:33:37 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:33:47 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:33:47 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:34:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 16:34:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 16:35:42 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 16:35:43 [ERROR] django.request: Internal Server Error: /api/token/refresh/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/auth_service/views.py", line 231, in post
    payload = decode_rsa_jwt(refresh_token)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/utils/jwt_rsa.py", line 112, in decode_rsa_jwt
    keypair = RSAKeyPair.objects.filter(kid=kid, active=True).first()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1104, in first
    for obj in queryset[:1]:
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 384, in __iter__
    self._fetch_all()
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 515, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 486, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/views.py", line 303, in post
    UserActivity.objects.create(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 663, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 902, in save
    self.save_base(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1008, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1169, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1210, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1868, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1882, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^

2025-09-15 16:39:09 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 16:56:57 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 17:18:02 [WARNING] django.request: Forbidden: /api/token/validate/
2025-09-15 17:18:02 [WARNING] django.request: Forbidden: /api/token/validate/
2025-09-15 17:18:10 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
2025-09-15 17:18:20 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:20 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:23 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:23 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:23 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:23 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:24 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:24 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:28 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:28 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:28 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:28 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:32 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:32 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:33 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:33 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:35 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:35 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:35 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:35 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:35 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:35 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:36 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:36 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:36 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:36 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:39 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:39 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:50 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:50 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:18:59 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:18:59 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:19:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:19:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:19:20 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:19:20 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:19:30 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:19:30 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:19:39 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:19:39 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:20:42 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:20:42 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:04 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:04 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:21 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:21 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:30 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:30 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:45 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:45 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:50 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:50 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:21:51 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:21:51 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:00 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:00 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:20 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:20 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:30 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:30 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:39 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:39 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:40 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:40 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:46 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:46 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:46 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:46 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:49 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:49 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:22:59 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:22:59 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:23:09 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:23:09 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:23:19 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:23:19 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:23:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:23:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:23:39 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:23:39 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:23:49 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:23:49 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:00 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:00 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:10 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:10 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:19 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:19 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:30 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:30 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:40 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:40 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:24:49 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:24:49 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:25:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:25:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:26:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:26:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:27:38 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:27:38 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:27:40 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:27:40 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:27:49 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:27:49 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:28:00 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:28:00 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:28:10 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:28:10 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:28:19 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:28:19 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:28:29 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:28:29 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:28:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:28:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:29:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:29:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:30:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:30:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:31:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:31:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:32:43 [DEBUG] core: Tenant from user: proliance
2025-09-15 17:32:43 [INFO] core: Retrieving tenant: 3 for tenant proliance
2025-09-15 17:33:43 [WARNING] django.request: Forbidden: /api/public-key/36aba0f0182245159f7a3c3a8cd62657/
2025-09-15 17:33:44 [ERROR] django.request: Internal Server Error: /api/token/refresh/
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/app/auth_service/views.py", line 231, in post
    payload = decode_rsa_jwt(refresh_token)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/utils/jwt_rsa.py", line 112, in decode_rsa_jwt
    keypair = RSAKeyPair.objects.filter(kid=kid, active=True).first()
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1104, in first
    for obj in queryset[:1]:
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 384, in __iter__
    self._fetch_all()
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1949, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 91, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1623, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_rsakeypair" does not exist
LINE 1: ...r"."updated_at", "users_rsakeypair"."active" FROM "users_rsa...
                                                             ^


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
psycopg2.errors.UndefinedTable: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/views/generic/base.py", line 105, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 515, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 475, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 486, in raise_uncaught_exception
    raise exc
  File "/usr/local/lib/python3.11/site-packages/rest_framework/views.py", line 512, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/auth_service/views.py", line 303, in post
    UserActivity.objects.create(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 663, in create
    obj.save(force_insert=True, using=self.db)
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 902, in save
    self.save_base(
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1008, in save_base
    updated = self._save_table(
              ^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1169, in _save_table
    results = self._do_insert(
              ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/base.py", line 1210, in _do_insert
    return manager._insert(
           ^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/query.py", line 1868, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/models/sql/compiler.py", line 1882, in execute_sql
    cursor.execute(sql, params)
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 122, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 79, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 92, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 100, in _execute
    with self.db.wrap_database_errors:
  File "/usr/local/lib/python3.11/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/usr/local/lib/python3.11/site-packages/django/db/backends/utils.py", line 105, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
django.db.utils.ProgrammingError: relation "users_useractivity" does not exist
LINE 1: INSERT INTO "users_useractivity" ("user_id", "tenant_id", "a...
                    ^

2025-09-15 17:34:27 [INFO] users: Login attempts reset for user support@prolianceltd.com in tenant proliance
